extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoLightHandler.java

```diff
@@ -81,7 +81,7 @@
     /**
      * The default refresh initial delay in Seconds.
      */
-    private static int DEFAULT_REFRESH_INITIAL_DELAY = 15;
+    private static int defaultRefreshInitialDelay = 15;
```

Comment:
constant, `private static final int DEFAULT_REFRESH_INITIAL_DELAY` please.

Reply:
It can not be a constant, because in [WemoLightHandlerOSGiTest](https://github.com/MusalaSoft/smarthome/blob/632e93ef832b30ea29cd0efed4ca77a09a237e11/extensions/binding/org.eclipse.smarthome.binding.wemo.test/src/main/groovy/org/eclipse/smarthome/binding/wemo/handler/test/WemoLightHandlerOSGiTest.groovy#L53) for the test we set it to 1 second.

Reply:
From a coding style pov the test must not access this field (since its `private` anyway, but only groovy doesn't care).
We could just drop the line from the test which will result in very slow execution (waiting 15s for each of the 7 test cases).
The other option would be to rework this and add a configuration property. @hmerk do you have any suggestion on how to deal with this issue? background: static analysis complains about this "variable" being all caps. And from the looks it should be a constant.

Reply:
Yes, it is a constant, therefore it is capitalized and somebody askedd me to change it a while ago. But if code analasys has an issue, we could remove it and hard code it in the according method.
I can put it on my long list of needed/wanted changes but can not tell when to have my updates ready.

Reply:
Thanks for the very quick reply @hmerk.
It was indeed introduced to override it during test execution. My suggestion is to revert it to a constant and remove the overriding in the test.
@kssimova could you go on with this? The tests will be super slow then but we should address that in a separate PR if needed.

Reply:
I agree, we should improve the test case later.
Perhaps we could also mitgrate it from Groovy to pure Java.